.circleci/config.yml

```diff
@@ -12,6 +12,7 @@ executors:
         environment:
           PGHOST: localhost
           PGUSER: ubuntu
+          COVERAGE: 1
```

Comment:
If this is just for boolean true, is there a more explicit literal we can use to be more, well, explicit about intent here? I know 0/1 as booleans _is_ well established convention, but where true and false literals are available I think it's clearer ie I'd never wonder if I could turn true into doubletrue ;)

Reply:
not really no. The 2 items you have commented on kind of go together - checking the presence means that the value here doesn't need to be truthy...

Reply:
Yeah it would, which is not a good thing in my opinion. You need more familiarity than I think you should in order to know that changing this to `"false"` or `false` or `0` would do nothing at all, which is my issue with the other bit.

And my issue with this bit is that having it as a 1 looks like it's numerical information, which is again needlessly opaque

Reply:
There are several ways of dealing with this, not sure which way is best? I've used several patterns for this in the past, none of them completely satisfactory. 

- Treat as an integer, with > 0 being enabled (so 0 turns it off) and calling to_i on the string (so nil becomes 0)
- This method with presence implying on
- check for literal string 'true'

WDYT? Is there a 4th option you think is better?

Reply:
We don't have the YAML translation layer here sadly - because yes you're right YAML false is false, but environment variable 'false' would come out true. So I'm not sure whether there is a 'best' option. Ruby doesn't have a 'to_boolean' method that I know of. 

There's ActiveRecord::Type::Boolean.new.cast(x) which converts lots of values into 'false', but we don't have rails loaded (present? doesn't work here) so it probably doesn't work either

Reply:
Ahhhh of course, yes that's what I was missing: that we were going from yaml to ruby, via env vars! 🤦 This is a good compromise solution!